pages/jobs.tsx

```diff
@@ -0,0 +1,70 @@
+// import Link from 'next/link'
+import { useState, useEffect } from 'react'
+import { GetStaticProps } from 'next'
+import { siteConfig } from '@/const/meta'
+import Content from 'components/Layout/Content'
+import { Section, Title, SubTitle, TitleSmall, LinkContainer, LoadingText } from 'const/styles/pages/content'
+import { ExternalLink } from '@/const/styles/global'
+import SVG from 'react-inlinesvg'
+
+async function getJobs() {
+  const { api, deptID } = siteConfig.greenhouse
+  const getJobs = await fetch(api)
+  const { jobs } = await getJobs.json()
+  const jobsData = []
+  await jobs.map((job) => {
+    const dept = job.departments[0]
+    if (dept) {
+      dept.id === deptID ? jobsData.push(job) : null
+    }
+  })
+
+  return jobsData
+}
+
+export default function Jobs({ jobsData }) {
+  const discordURL = siteConfig.social.discord.url
+  const [jobs, setJobs] = useState(jobsData)
+  const [isLoading, setLoading] = useState(false)
+
+  // Fetch the latest jobs to possibly override static rendered (getStaticProps) SSR jobs
```

Comment:
why would you need to do this? 

I mean, the `getStaticProps` should work fine. Specially if you add the revalidate option. The only thing, they will have a delay in order to be shown. but it will be perforrmant and live enough. right?

Reply:
Again, my current understanding of the revalidation option isn't sufficient to know if and how revalidation happens on server side. It seems like this is a so called 'serverless' function in this case supported by Vercel. I'll remove the client side update in this case and we can test.